lib/rvg/describable.rb

```diff
@@ -11,7 +11,7 @@ class RVG
     module Describable
       private
 
-      def initialize(*args, &block) #:nodoc:
+      def initialize(*args, &block)
```

Comment:
@Watson1978 did you intend for this to become public?

Reply:
Since there is a `Module#private` right above, so yard treats it as a private method.